### ISSUE TITLE
gbdxtools module - ipython notebook is html, not json

### ISSUE THREAD

**Issue by joemcglinchy** [GeoBigData/gbdx-training#6]
Hello,

I cannot open the notebook  gbdxtools-tutorial.ipynb because it is html. I also tried renaming the extension to html but that doesn't load properly, either.

![image](https://user-images.githubusercontent.com/4762214/29465514-d2144e08-83f6-11e7-86e5-e1fd4019e345.png)

**Comment by dangetman** [comment]
Hi @joemcglinchy 

You will need to run this notebook using the jupyter notebook server. Have a look at this section of the instructions and let us know if you are having trouble running that in your environment.

**Comment by joemcglinchy** [comment]
Hi @dangetman 

The screenshot I provided is running it from the jupyter notebook server. Unless that is not correct? I ran the command `jupyter notebook` from the directory where I downloaded the pynb file. Please advise.

**Comment by dangetman** [comment]
Apologies, I thought you were attempting to run this by changing the extension to HTML. 

I haven't seen this issue, but it looks like the file is corrupted. Was this pulled from git or downloaded in some other way?

**Comment by joemcglinchy** [comment]
No worries, I was attempting to run it by changing the extension to HTML but only after seeing the jupyter notebook gave me an error message when trying to run it that way. I saw the first few lines of code in the .ipynb file were HTML so I thought I would give that a shot.

It was not pulled from git. I downloaded it via right click > save as from this webpage:

https://github.com/GeoBigData/gbdx-training/blob/master/gbdxtools_module/README.md 

at section 4.1.
